javascripts/map.js

```diff
@@ -1519,8 +1519,11 @@ function printDirections(){
 
 //determine which stylesheet to use, mobile or desktop
 function isMobile() {
-  try{ document.createEvent("TouchEvent"); return true; }
-  catch(e){ return false; }
+	if (navigator.userAgent.match(/Mobi/))
```

Comment:
Greg specifically wanted printing options. The main reason behind the print buttons was to allow for printing the map vs printing directions, and regular ctrl+p printing sometimes has issues